config/deploy.rb

```diff
@@ -43,5 +43,12 @@
       execute "cd #{current_path} && ./gradlew jar"
     end
   end
+
+  task :install_dist do
+    on roles(:app) do
+      execute "cd #{current_path} && ./gradlew build installDist"
+    end
+  end
 end
 after 'deploy:finished', 'gradle:jar'
+after 'deploy:finished', 'gradle:install_dist'
```

Comment:
Hmmm...I was just wondering if for development purposes it might be worth leaving the ability to build a jar file in there? Like if we're doing some hacking somewhere? Happy to take it out.